packages/eos-components/src/components/WelcomeCard.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+  <b-card
+    :title="title"
+    :subTitle="subTitle"
+    :borderVariant="selected ? 'primary' : 'default'"
+    :class="{ active: selected, disabled: disabled }"
+    titleTag="h6"
+    subTitleTag="h4"
+    imgBottom
+    :imgSrc="img"
+  >
+    <!-- eslint-disable vue/no-v-html -->
+    <p class="text-muted">
+      {{ text }} <br v-if="secondaryText"> {{ secondaryText }}
```

Comment:
I think we should figure this out with Joana. Have you asked her?

Reply:
I didn't ask, It was clear for me that it's two text blocks, but in any case, we can ask and the change will be easy to apply.

Reply:
Yes, we are hardcoding right now but it would be great to move this metadata to the manifest file. So if the manifest file format needs a title, subtitle and yet another sub-subtitle it would be great to know in advance. And please don't use HTML in those fields.